charts/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.8.10
+version: 5.8.11
```

Comment:
Actually the version shall be 5.8.13 or larger.

Reply:
Done